bin/generate_lts_report.py

```diff
@@ -67,7 +68,8 @@ def detect_baseline(build_result, builds_url):
 
 
 def get_build_report(project_url, unfinished=False,
-                     baseline=None, build_id=None):
+                     baseline=None, build_id=None,
+                     timeout=60):
```

Comment:
I would give it a bit more time. In theory there is a separate queue for handling these, but I'm not sure how the task scheduling works in detail. Maybe 2 min is not too long?

Reply:
I changed to 120. Thanks.